homeassistant/components/zwave/util.py

```diff
@@ -1,54 +1,71 @@
 """Zwave util methods."""
 import logging
 
+from . import const
+
 _LOGGER = logging.getLogger(__name__)
 
 
-def value_handler(value, method=None, class_id=None, index=None,
-                  label=None, data=None, member=None, instance=None,
-                  **kwargs):
-    """Get the values for a given command_class with arguments.
+def check_node_schema(node, schema):
+    """Check if node matches the passed node schema."""
+    if (const.DISC_NODE_ID in schema and
```

Comment:
It's something I considered, and I could go either way on it. It would mean schema definitions would be a little more verbose, since almost everything would include `vol.In(` wrapping the requirements. We would then gain some more flexibility though if we wanted to do more advanced selection later.